File: DaS.ScriptLib/Game/Data/Structures/WIP/MSB.vb
Imports DaS.ScriptLib.Injection

Namespace Game.Data.Structures

    Public Class MSB

        Public Shared ReadOnly Property Pointer As Integer
            Get
                Return RInt32(&H13785A0)
            End Get
        End Property

        Public Shared ReadOnly Property FirstEntryPointer As Integer
            Get
                Return RInt32(Pointer + &HC)
            End Get
        End Property

        Public Shared ReadOnly Property Entries(i As Integer)
            Get
                Return "fuck off vb"
            End Get
        End Property

        Private Shared Sub ReadEntries()



        End Sub

    End Class

End Namespace

File: DaS.ScriptLib.NEW/Game/Data/Structures/WIP/MSB.cs
using DaS.ScriptLib.Injection;

namespace DaS.ScriptLib.Game.Data.Structures
{

    public class MSB
	{

		public static int Pointer {
			get { return Hook.RInt32(0x13785a0); }
		}

		public static int FirstEntryPointer {
			get { return Hook.RInt32(Pointer + 0xc); }
		}

		public static object Entries {
			get { return "fuck off vb"; }
		}


		private static void ReadEntries()
		{


		}

	}

}
